scripts/remove-features.mjs

```diff
@@ -164,16 +156,16 @@ const performRemovals = async () => {
   
   if (!keepFeatures.includes("schedule-manager")) {
```

Comment:
this area could get tricky to maintain in the future as we add in serverless packages but we can probably refactor this slightly later.